hugegraph-core/src/main/java/com/baidu/hugegraph/auth/StandardAuthManager.java

```diff
@@ -516,10 +532,228 @@ public UserWithRole validateUser(String token) {
         return new UserWithRole(user.id(), username, this.rolePermission(user));
     }
 
+    @Override
+    public Id createProject(HugeProject project) {
+        E.checkArgument(!Strings.isNullOrEmpty(project.name()),
+                        "The name of project can't be null or empty");
+        return commit(() -> {
+            // Create project admin group
+            if (project.adminGroupId() == null) {
+                HugeGroup adminGroup = new HugeGroup(project.adminGroupName());
+                adminGroup.creator(project.creator());
+                Id adminGroupId = this.createGroup(adminGroup);
+                project.adminGroupId(adminGroupId);
+            }
+
+            // Create project op group
+            if (project.opGroupId() == null) {
+                HugeGroup opGroup = new HugeGroup(project.opGroupName());
+                opGroup.creator(project.creator());
+                Id opGroupId = this.createGroup(opGroup);
+                project.opGroupId(opGroupId);
+            }
+
+            // Create project target to verify permission
+            HugeResource resource = new HugeResource(ResourceType.PROJECT,
+                                                     project.name(),
+                                                     null);
+            HugeTarget target = new HugeTarget(project.targetName(),
+                                               this.graph.name(),
+                                               "localhost:8080",
+                                               ImmutableList.of(resource));
+            target.creator(project.creator());
```

Comment:
ditto

Reply:
"creator" is a parameter. I see that when other places are created, the method "AuthManagerProxy.updateCreator" is called, so to be consistent, I also update "creator" when I create it.